DBM-ChamberOfAspects/Obsidian/Sartharion.lua

```diff
@@ -29,9 +29,10 @@ mod:AddBoolOption("AnnounceFails", true, "announce")
 
 local timerShadowFissure    = mod:NewCastTimer(5, 59128)--Cast timer until Void Blast. it's what happens when shadow fissure explodes.
 local timerWall             = mod:NewCDTimer(30, 43113)
-local timerTenebron         = mod:NewTimer(30, "TimerTenebron", 61248)
-local timerShadron          = mod:NewTimer(80, "TimerShadron", 58105)
-local timerVesperon         = mod:NewTimer(120, "TimerVesperon", 61251)
+local timerTenebron         = mod:NewTimer(33, "TimerTenebron", 61248)
+local timerShadron          = mod:NewTimer(120, "TimerShadron", 58105)
+local timerVesperon         = mod:NewTimer(80, "TimerVesperon", 61251)
+local enrageTimer			= mod:NewBerserkTimer(720)
 
```

Comment:
For those drake timers to actually take effect You need to change we handle it in function CheckDrakes